Yoast/wordpress-seo
# Filtered for prefix dashes

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Used if statement to filter the dashes. Changed what is sent to the function to only be the slug. The baseurl after the protocol is removed has been sent along in a context variable. To be able to check where the user submitted slug begins.

## Test instructions

This PR can be tested by following these steps:

* Check if the unit tests pass on wordpress-seo.
* Checkout this branch and it's sibling branch: https://github.com/Yoast/yoast-components/pull/571
* Visit a post page, add a url with a prefix space and a trailing space, verify no dashes are shown in the snippet preview.

Hyphens typed by the user will unfortunately also be filtered, this has been discussed, and is not considered a big enough problem to stop this pull.

## Quality assurance


Fixes #9892
